test/functionalTest/cases/0000_deprecated_checkings/metadata_id_duplicate_error.test

```diff
@@ -108,9 +108,9 @@ Date: REGEX(.*)
                 "type": "T"
             },
             "statusCode": {
-                "code": "422",
+                "code": "207",
```

Comment:
This shouldn't be 2xx as there is no partial success in this case for the creation of an entity with duplicated attributes.

(NOTE: probably 422 is also wrong as according to "Error Response" rules at https://fiware.github.io/specifications/ngsiv2/stable this is a case in which the error is caused by request itself (i.e. they do not depend on the NGSIv2 server status) so 400 should be used instead. But this is a lateral issue)